### PR TITLE
Feat: Substrate update people who stake + notification

### PR DESCRIPTION
**Pull request by SHA888**
## JIRA link
https://blocksphere2020.atlassian.net/browse/DBIO-545?atlOrigin=eyJpIjoiYmVlMTdhMjlkMjEyNDBiMGE3ZGM2N2M5ZjZkMjJjZjYiLCJwIjoiaiJ9

## Changelog / Description
Update substrate on list of people who stake + notification

- Add Stakers trait in user-profile pallet
- Add `stakers by account id getter` and `setter` fn in user-profile pallet
- Add `StakersByAccountId` storage in user-profile pallet

### CODE REVIEW

**Review by kevinjanada — CHANGES_REQUESTED** [review]
Please refer to JIRA Card

**Comment by SHA888** [comment]
Noted mas. Thank you

File: pallets/genetic-testing/src/lib.rs [hunk 1]
```diff
@@ -65,6 +65,8 @@ pub mod pallet {
         DnaSampleResultReady(DnaSampleOf<T>),
         /// Dna Test Result Submitted
         DnaTestResultSubmitted(DnaTestResultOf<T>),
+        /// Submit Data Staking Details
+        DataStaked(HashOf<T>)
```
Comment (kevinjanada) [inline pallets/genetic-testing/src/lib.rs:69]: di dalem DataStaked minta tolong ditambah AccountId si staker nya yah bli

File: pallets/genetic-testing/src/lib.rs [hunk 2]
```diff
@@ -537,6 +560,28 @@ impl<T: Config> GeneticTestingInterface<T> for Pallet<T> {
...
+    {
+        let data_staker = data_staker.clone();
+        if data_staker != data_staker.clone() { 
+            return Err(Error::<T>::DataStakerNotFound);
+        }
+
+        let data_hash = data_hash.clone();
+        if data_hash != data_hash.clone() { 
+            return Err(Error::<T>::DataHashNotFound);
+        }
```
Comment (kevinjanada) [inline pallets/genetic-testing/src/lib.rs:579]: Bli @SHA888 , kliatannya ini gak perlu bli, dia hanya membandingkan dengan variable itu sendiri.
Dan untuk nge submit ini memang harusnya gak perlu ada validasi

**Review by kevinjanada — CHANGES_REQUESTED** [review]

**Comment by SHA888** [comment]
Mas @kevinjanada sudah aku perbaikin ya mas. Tolong dicek kembali. Thank you

**Review by kevinjanada — APPROVED** [review]
Looks good

**Pull request merged**